---
title: MSSQLSERVER_8630 | Документация Майкрософт
ms.custom: ''
ms.date: 04/04/2017
ms.prod: sql
ms.prod_service: database-engine
ms.service: ''
ms.component: errors-events
ms.reviewer: ''
ms.suite: sql
ms.technology:
- database-engine
ms.tgt_pltfrm: ''
ms.topic: language-reference
helpviewer_keywords:
- 8630 (Database Engine error)
ms.assetid: 45a9d084-b5ad-435b-ac19-5f4352515817
caps.latest.revision: 14
author: edmacauley
ms.author: edmaca
manager: craigg
ms.workload: Inactive
ms.openlocfilehash: a0e8144373bf120c1c2a0d88616d72eedc27fd4a
ms.sourcegitcommit: 7a6df3fd5bea9282ecdeffa94d13ea1da6def80a
ms.translationtype: HT
ms.contentlocale: ru-RU
ms.lasthandoff: 04/16/2018
---
# <a name="mssqlserver8630"></a>MSSQLSERVER_8630
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
  
## <a name="details"></a>Сведения  
  
|||  
|-|-|  
|Название продукта|SQL Server|  
|Идентификатор события|8630|  
|Источник события|MSSQLSERVER|  
|Компонент|SQLEngine|  
|Символическое имя|EXECUTION_ERR|  
|Текст сообщения|внутренняя ошибка обработчика запросов: обработчик запросов обнаружил непредвиденную ошибку во время выполнения.|  
  
## <a name="explanation"></a>Объяснение  
Внутренняя ошибка обработчика запросов.  
  
## <a name="user-action"></a>Действие пользователя  
Обратитесь к основному поставщику услуг. Предоставьте информацию из этого сообщения об ошибке.  
  
